### ISSUE TITLE
Keras version problem

### ISSUE THREAD

**Issue by superguopeng** [entron/entity-embedding-rossmann#19]
I'm sorry I asked such a question,I used keras2.2.4,It was found  'Merge' that it could not be introduced in 'keras.layers.core ',But I found 'merge' in 'keras.layers',Unfortunately, the following error occurred
Thank you very much !
![image](https://user-images.githubusercontent.com/41366948/59399594-2a591400-8dc7-11e9-8c1f-a23143a7bffa.png)

**Comment by entron** [comment]
I haven't used keras for sometime, from your error it seems keras changed the API for Reshape layer. Could you try to change the parameter  name from `dim` to `target_shape`?

**Comment by superguopeng** [comment]
Thank you very much，I'll try your solution

**Comment by wmmxk** [comment]
> I haven't used keras for sometime, from your error it seems keras changed the API for Reshape layer. Could you try to change the parameter name of Reshape from `dim` to `target_shape`?

Do you still remember what version of Keras you used?

**Comment by entron** [comment]
I  tested it with keras 2.1.5

**Comment by entron** [comment]
@superguopeng It would be great if you can create a pull request if you make it work with the latest keras.

**Comment by entron** [comment]
Tested the code with latest Keras (2.3.1) it seems everything is working fine. Therefore, I will close this issue.
